src/inmanta/server/services/databaseservice.py

```diff
@@ -100,21 +109,44 @@ async def get_status(self) -> DataBaseReport:
         """Get the status of the database connection"""
         connected = await self.get_connection_status()
 
+        if not connected:
+            return DataBaseReport(
+                connected=connected,
+                database=self.dn_name,
+                host=self.db_host,
+                max_pool=0,
+                free_pool=0,
+                open_connections=0,
+                free_connections=0,
+                pool_exhaustion_time=self._exhausted_pool_events_count * self._db_exhaustion_check_interval,
+            )
+
+        pool = get_pool()
+
         return DataBaseReport(
             connected=connected,
             database=self.dn_name,
             host=self.db_host,
-            max_pool=self._pool.get_max_size(),
-            free_pool=self.get_pool_free(),
-            open_connections=self._pool.get_size(),
-            free_connections=self._pool.get_idle_size(),
+            max_pool=pool.size() + self._max_overflow,
+            free_pool=pool.checkedin(),
```

Comment:
```suggestion
            free_pool=pool.get_free_connections(pool),
```

if I look at the docs: number of connections not in use in the pool

(also below)